fabfile.py

```diff
@@ -71,6 +71,30 @@ def launch_snapshot(c, snapshot_id):
     print(f"The instance id is '{result['instance_id']}'")
     print(f"The instance name is '{result['instance_name']}'\n")
 
+# Terminate an AWS instance.
+@task
+def terminate(c, instance_id=None, instance_name=None):
+    if not instance_id and not instance_name:
+        print("Missing arguments, please use the --help option to read about the command.")
+        return
+
+    if instance_id and instance_name:
+        print("Only specify one argument, please use the --help option to read about the command.")
+        return
+
+    if instance_name is not None:
+        if not instance_name:
```

Comment:
this would be true if instance_name is not none, do you mean to check after you try to get instance id?

Reply:
I was trying to catch the empty string which is not None but still evaluates to false.

Reply:
As discussed: 00e00a1035556fbe3b248eac8973de7f74d9f51a